sdk/storage/azure-storage-files-shares/src/rest_client.cpp

```diff
@@ -2436,7 +2436,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
         {
           kUnknown,
           kEnumerationResults,
-          kHandleList,
+          kEntries,
```

Comment:
It's usually not a good idea to add new features and fix bugs in a single PR. Since it's not targeting `main` branch, I'll let it go this time.

Don't forget to update the changelog.

Reply:
Ok, I got it. I will split into two PRs next time.